doc/hardware/emulator/adc_emul.rst

```diff
@@ -0,0 +1,24 @@
+.. _adc_emul_api:
+
+
+ADC Emulator
+###################################
+
+Overview
+********
+The ADC emulator provides the ability to be perform tests without access to
```

Comment:
`index.rst` already says at the beginning that emulators support testing without access to the real hardware. I think having the same sentence in the page for each specific emulator is redundant. We should probably be able to say something more interesting here, but I don't think that needs to block this PR.

If you want an initial blurb for each emulator, a brief description of the type of hardware it emulates seems like a reasonable choice, e.g. "Analog-to-digital converters allow the system to read external analog signals. The ADC emulator simulates such a device."

Reply:
Force-pushed the changes to improve the information in each *_emul file.